tests/sqlparser_common.rs

```diff
@@ -6387,6 +6394,46 @@ fn parse_create_or_replace_view() {
             assert!(!materialized);
             assert!(or_replace);
             assert_eq!(cluster_by, vec![]);
+            assert!(comment.is_none());
+            assert!(!late_binding);
+            assert!(!if_not_exists);
+            assert!(!temporary);
+        }
+        _ => unreachable!(),
+    }
+}
+
+#[test]
```

Comment:
we can put the test in [sqlparser_snowflake.rs](https://github.com/sqlparser-rs/sqlparser-rs/blob/33963675d75c11e64e423fd79ab998193dbb4286/tests/sqlparser_snowflake.rs)